src/analytics/api/views/data.py

```diff
@@ -114,39 +114,52 @@ def post(self):
 
         postfix = "-" if output_format == "airqo-standard" else "-aqcsv-"
 
-        data_frame = EventsModel.download_from_bigquery(
-            sites=sites,
-            devices=devices,
-            airqlouds=airqlouds,
-            start_date=start_date,
-            end_date=end_date,
-            frequency=frequency,
-            pollutants=pollutants,
-        )
-
-        if data_frame.empty:
-            return (
-                create_response("No data found", data=[]),
-                Status.HTTP_404_NOT_FOUND,
+        try:
+            data_frame = EventsModel.download_from_bigquery(
+                sites=sites,
+                devices=devices,
+                airqlouds=airqlouds,
+                start_date=start_date,
+                end_date=end_date,
+                frequency=frequency,
+                pollutants=pollutants,
             )
 
-        records = data_frame.to_dict("records")
+            if data_frame.empty:
+                return (
+                    create_response("No data found", data=[]),
+                    Status.HTTP_404_NOT_FOUND,
+                )
```

Comment:
@Codebmk, was is a request to return an empty file instead of `No data found`. What is your view?? cc @Baalmart